runtime/platform/compiler.h

```diff
@@ -100,22 +100,38 @@
 #endif // (__cplusplus) >= 202002L
 
 /// Define a C symbol with weak linkage.
+#ifdef _MSC_VER
+// There currently doesn't seem to be a great way to do this in Windows and
+// given that weak linkage is not really critical on Windows, we'll just leave
+// it as a stub.
+#define ET_WEAK
```

Comment:
Hi, we are adding `Windows x64 Build Support` https://github.com/pytorch/executorch/pull/6979. I use `.\install_requirements.bat --pybind xnnpack` to build on windows. And we encountered duplicate symbol problem. 
![image](https://github.com/user-attachments/assets/f3a3ca3d-b3ad-4d3a-8691-5abf120404dc)
I investigated and found that windows didn't support weak symbol. Do you have some insights how to resolve this?